onnx/onnx.proto.in

```diff
@@ -345,6 +348,15 @@ message SparseTensorProto {
 message ValueProto {
 
   message KeyValuePairProto {
+    enum KeyType {
+      UNDEFINED = 0;
+      STRING = 1;
+      INT32 = 2;
+      INT64 = 3;
+      UINT64 = 4;
+    }
+    // This field MUST be present for this version of the IR.
+    optional KeyType type = 5;
```

Comment:
We don't need the enum in this case, as we have already used oneof there. There will be a "enum" generated with oneof usage.